src/Tools/NodeDocumentationMarkdownGenerator/CommandHandler.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Text;
+using Dynamo.Logging;
+using NodeDocumentationMarkdownGenerator.Commands;
+using NodeDocumentationMarkdownGenerator.Verbs;
+
+namespace NodeDocumentationMarkdownGenerator
+{
+    internal static class CommandHandler
+    {
+        internal static string HandleFromPackage(FromPackageOptions opts)
+        {
+            try
+            {
+                FromPackageFolderCommand.HandlePackageDocumentation(opts);
+            }
+            catch (Exception e)
+            {
+                LogExceptionToConsole(e);
+            }
+
+            return "";
+        }
+
+        internal static string HandleFromDirectory(FromDirectoryOptions opts)
+        {
+            try
+            {
+                FromDirectoryCommand.HandleDocumentationFromDirectory(opts);
+            }
+            catch (Exception e)
+            {
+                LogExceptionToConsole(e);
+            }
+
+            return "";
+        }
+
+        internal static void LogExceptionToConsole(Exception e)
+        {
+            var strBuilder = new StringBuilder();
+            strBuilder.AppendLine($"{e.GetType()} :");
+            strBuilder.AppendLine(e.Message);
+            strBuilder.AppendLine(e.StackTrace);
+            Console.WriteLine(strBuilder.ToString());
+        }
+    }
+
+    internal class DummyConsoleLogger : ILogger
```

Comment:
summary would be good - something like - `A logger implementation used to construct Dynamo core model types`.